DrinkACupOfJuice/Q&A2.0/Q&A.js

```
@@ -0,0 +1,249 @@
+                const questionList = document.getElementById('questionList');
+                questionList.innerHTML = ''; // 清空问题列表的内容
+
+                if (Array.isArray(data) && data.length > 0) {
+                    data.forEach(questionObj => {
+                        if (questionObj.hasOwnProperty('id') && questionObj.hasOwnProperty('title')) {
+                            // 创建一个新的div来包裹每个问题，并添加到问题列表中
+                            const questionDiv = document.createElement('div');
+
+                            questionDiv.style.backgroundColor = 'white'; // 与页面主题一致的浅绿色背景
+                            questionDiv.style.padding = '10px'; // 内边距
+                            questionDiv.style.border = '1px solid #008000'; // 深绿色边框
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)